stdlib/public/core/CTypes.swift

```diff
@@ -201,6 +201,27 @@ extension OpaquePointer : Equatable {
 }
 
 /// The corresponding Swift type to `va_list` in imported C APIs.
+#if arch(arm64) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS))
```

Comment:
Please add a comment here describing this as the AAPCS64 case.

Reply:
Sure